platform/linuxbsd/power_linuxbsd.cpp

```diff
@@ -1,5 +1,5 @@
 /*************************************************************************/
-/*  power_x11.cpp                                                        */
+/*  power_linuxbsd.cpp                                                   */
```

Comment:
I'm not sure this code even works on BSDs. FreeBSD only has /proc for Linux compatibility iirc?

Reply:
Weather it works or not, it *is* used on all X11 platforms (as the old name suggests). Therefor, I think this is an appropriate name.

Reply:
It:

1. Works on non-X11 displays, like Wayland
2. Doesn't *break* on BSD
3. Is already somewhat generic and adding BSD support would be easy

So I agree with @wmww that the name here is right.